src/main/java/nl/dtls/fairdatapoint/api/config/ApplicationFilter.java

```diff
@@ -61,14 +61,20 @@
     public void doFilterInternal(final HttpServletRequest request,
             final HttpServletResponse response, final FilterChain fc)
             throws IOException, ServletException {
-        String allowedMtds = (RequestMethod.GET.name() + ","
-                + RequestMethod.POST.name() + "," + RequestMethod.PATCH.name());
         
+        StringBuilder allowedMtds = new StringBuilder();
+        allowedMtds.append(RequestMethod.GET.name());
+        allowedMtds.append(",");
+        allowedMtds.append(RequestMethod.POST.name());
+        allowedMtds.append(",");
+        allowedMtds.append(RequestMethod.PATCH.name());
```

Comment:
For ease of maintenance we might better use [String.join()](https://docs.oracle.com/javase/8/docs/api/java/lang/String.html#join-java.lang.CharSequence-java.lang.CharSequence...-).